### DISCUSSION TITLE
Self-Hosted: Possible to increase file size limits?

### DISCUSSION THREAD

**Discussion by Handrail9** [ente-io/ente#2215] [Q&A] [answered]
Hi, I have a few videos that are larger than 4GB, is it possible to upload them to my self hosted instance at this time?
Thanks

**Comment by mnvr** [comment, marked as answer]
You can search in the code for "maxFileSize" - you'll find three places (two on the clients, and one on server) where this value is set. You can change it for your self hosted instance. I'm not sure, but I think these are the only three places to change.

Note: Goes without saying, but you're moving into uncharted territory if you change your self hosted instance to beyond the supported file sizes.

**Reply by Handrail9** [reply]
Hello, thanks for the response. Hoping that eventually it gets tested and supported enough to be an env variable, but I may give it a try soon and report back if theres any issues or goes smoothly

**Comment by fvnky07** [comment]
why is there a limit? i have videos that are multiple tens of gbs. This is almost a dealbreaker for me. I understand for free users but paid users?
